services/state/contract/storage_slot.proto

```diff
@@ -0,0 +1,70 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2023 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import "state/common.proto";
+
+option java_package = "com.hederahashgraph.api.proto.java";
+// <<<pbj.java_package = "com.hedera.hapi.node.state.contract">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+/**
+ * The key of a storage slot. A slot is scoped to a specific contract number.
+ * 
+ * For each contract, its EVM storage is a mapping of 256-bit keys (or "words") to 256-bit values. 
+ */
+message SlotKey {
+    /**
+     * The number of the contract whose storage this slot belongs to.
+     */
+    int64 contract_number = 1;
+
+    /**
+     * The EVM key of this slot, when left-padded with zeros to form a 256-bit word.
+     */
+    bytes value = 2;
```

Comment:
So this single thing in one message is called all of "Slot", "key", and "value".  Three names for one thing.  ("Slot" and "key" together in the message name, "slot" in the comment @L43, and "value" is the name of the field.) 

(I get the distinction between the "slot" which holds the value and the "key" which identifies the "slot" - but I think in practice this distinction is glossed over and both the "slot" and the "key" which refers to it are called "slot".  Maybe not though.)

I guess minimum fix - and acceptable - would just be to change "value" -> "key"?

Reply:
Renamed this.